src/components/templates/Playa/MyAvatar.tsx

```diff
@@ -55,6 +53,15 @@ export const MyAvatar: React.FunctionComponent<PropsType> = ({
       setState((state) => {
         if (state) {
           let needsUpdate = false;
+          // Work around possible bad state that can happen in the presence of scroll jank
+          if (pressedKeys["ArrowLeft"] && pressedKeys["ArrowRight"]) {
+            pressedKeys["ArrowLeft"] = false;
+            pressedKeys["ArrowRight"] = false;
+          }
+          if (pressedKeys["ArrowUp"] && pressedKeys["ArrowDown"]) {
+            pressedKeys["ArrowUp"] = false;
+            pressedKeys["ArrowDown"] = false;
+          }
```

Comment:
why is this necessary?

Reply:
Janky scrolling while I was messing with keys. Then the booleans got stuck because the key release events didn't fire.